internal/clientcache/internal/cache/status.go

```diff
@@ -84,8 +90,22 @@ func (s *StatusService) Status(ctx context.Context) (*Status, error) {
 	}
 	for _, u := range users {
 		us := UserStatus{
-			Id:      u.Id,
-			Address: u.Address,
+			Id: u.Id,
+		}
+
+		{
```

Comment:
Any reason to have this be in a block? It's part of populating the user status struct above and it's already in a block (the for loop body).